pymc3/distributions/distribution.py

```diff
@@ -476,8 +476,14 @@ class Bound(object):
 
     Example
     -------
-    boundedNormal = pymc3.Bound(pymc3.Normal, lower=0.0)
-    par = boundedNormal(mu=0.0, sd=1.0, testval=1.0)
+    # In general Bounded distribution need to be defined befor model context
```

Comment:
I've defined them inside a context before. I don't think it matters (it shouldn't). 

Also, before has an "e" at the end.